runtime/calamari/tests/integrations_mock/mod.rs

```diff
@@ -19,14 +19,42 @@
 pub mod integration_tests;
 pub mod mock;
 
+use calamari_runtime::opaque::SessionKeys;
 pub use calamari_runtime::{currency::KMA, Event, Origin, Runtime, System};
 
 use frame_support::weights::{DispatchInfo, Weight};
-use manta_primitives::types::Balance;
+use lazy_static::lazy_static;
+use manta_primitives::types::{AccountId, Balance};
+use session_key_primitives::util::{unchecked_account_id, unchecked_collator_keys};
+use sp_core::sr25519::Public;
+#[cfg(feature = "std")]
+pub(crate) use std::clone::Clone;
 
-pub const BOND_AMOUNT: Balance = 1_000 * KMA;
+pub const COLLATOR_MIN_BOND: Balance = 4_000_000 * KMA;
```

Comment:
Seems both bond values can be imported from `staking.rs`.

Reply:
good call, thanks!